benchmark/scripts/run_smoke_bench

```diff
@@ -170,35 +165,27 @@ def merge(results, other_results):
     return results
 
 
-def test_performance(opt_level, old_dir, new_dir, threshold, num_samples,
-                     output_file):
+def test_performance(opt_level, old_dir, new_dir, threshold, output_file):
     """Detect performance changes in benchmarks.
 
-    Start fast with few samples per benchmark and gradually spend more time
-    gathering more precise measurements of the change candidates.
+    Gather more independent measurements of the change candidates.
     """
-
-    i, unchanged_length_count = 0, 0
+    i = 0
     old, new = [BenchmarkDriver(DriverArgs(dir, optimization=opt_level))
                 for dir in [old_dir, new_dir]]
     results = [measure(driver, driver.tests, i) for driver in [old, new]]
     tests = TestComparator(results[0], results[1], threshold)
-    changed = tests.decreased + tests.increased
+    changed = tests.decreased + tests.increased + tests.added
 
-    while len(changed) > 0 and unchanged_length_count < 10:
+    while len(changed) > 0 and i < 16:
```

Comment:
The last 3 commits here were done from mobile and were my attempt to somehow address the apparent instability by reverting to the old brute force methods. I'm pretty sure that my original approach to stop at 10 measurements was a correct one, but I need to handle the special cases of benchmarks with runtimes under 20 μs as a special case in result comparison and also adding filtering for min outliers like is the case with `ObjectiveCBridgeStubToNSDateRef`.

Reply:
![IMG_0747](https://user-images.githubusercontent.com/1046899/62961273-c077dd80-bdfc-11e9-90b6-d0bd6e764bfe.jpeg)

Reply:
Another example of benchmark that jumped up here during the first run, which seems like something that has about 2:10 chance to look unstable is we stick to the naive comparison of MIN values is `DataAppenDataSmallToSmall`: 
![IMG_0745](https://user-images.githubusercontent.com/1046899/62961580-527fe600-bdfd-11e9-9e8a-899855794ff5.jpeg)

Reply:
I’d like to fix that issue by expanding on the dubious indicator **(?)** in the ratio column, so that it revealed more details about the underlying sample distribution, to enable us to make more informed decisions in such cases.